app/services/yelp_service.rb

```diff
@@ -0,0 +1,25 @@
+class YelpService
+
+  def self.get_restaurant(food_type, lat, lon)
+    response = conn.get("businesses/search?term=#{food_type}&latitude=#{lat}&longitude=#{lon}")
+    parse = JSON.parse(response.body, symbolize_names: true)
+    # require 'pry';binding.pry
+    munchie_data = []
+    # parse[:businesses].each do |b|
```

Comment:
Still unsure of how to get this to work without over writing the previous instance each time. I hope this works for the assessment but maybe I can work on implementing it when I'm done with the rest of the project 😅